docs/exts/templates/deprecations.rst.jinja2

```diff
@@ -0,0 +1,37 @@
+{#
```

Comment:
Question...
is it possible to also have a version of this template per provider in the specific provider package?
so that Amazon deprecations will be shown in the Amazon provider directly
(Maybe is a future enhancement in a followup PR?)

Reply:
Yeah.  This looks like a follow up :)

Reply:
I think it's a great idea. For me, the closer we get to the actual code docs with the deprecation information, the better :)